### PR TITLE
Akhilesh latest us2221- Latest commits and code fixes for MarketWatch, WSJ, FN, Barrons and MG Preference Center page, authors, following frequency and element IDs updated

### PR DESCRIPTION
**Pull request by AkhileshKhanduri25**
Latest commits and code fixes for MarketWatch, WSJ, FN, Barrons and MG Preference Center page, authors, following frequency and element IDs updated

### CODE REVIEW

File: panda-qa/Resources/PageObjects/MarketWatchMailboxPage.robot
Comment (bhoomraogaripriyanka) [inline panda-qa/Resources/PageObjects/MarketWatchMailboxPage.robot:1]: @AkhileshKhanduri25 , Line 81 is not working for me.

File: panda-qa/Resources/PageObjects/MarketWatchPreferenceCenterPage.robot
Comment (bhoomraogaripriyanka) [inline panda-qa/Resources/PageObjects/MarketWatchPreferenceCenterPage.robot:1]: @AkhileshKhanduri25 ,please execute this test case as I am unable to navigate till this page

File: panda-qa/Resources/PageObjects/WSJAuthorsPage.robot
Comment (bhoomraogaripriyanka) [inline panda-qa/Resources/PageObjects/WSJAuthorsPage.robot:1]: @AkhileshKhanduri25 ,I am unable to execute the testcase ,it is getting failed while signing in,please verify from your end

File: panda-qa/Resources/PageObjects/WSJPreferenceCenterPage.robot
Comment (bhoomraogaripriyanka) [inline panda-qa/Resources/PageObjects/WSJPreferenceCenterPage.robot:1]: @AkhileshKhanduri25 ,I am unable to execute the testcase ,it is getting failed while signing in,please verify from your end

File: panda-qa/Resources/PageObjects/MarketWatchGlobalCapabilitiesPage.robot
Comment (bhoomraogaripriyanka) [inline panda-qa/Resources/PageObjects/MarketWatchGlobalCapabilitiesPage.robot:1]: @AkhileshKhanduri25 , Please provide the testcase/ suite name where you have executed MWCapabilities , I am unable to find where capabilities page is used

File: panda-qa/Resources/PageObjects/PENCapabilitiesPage.robot
Comment (bhoomraogaripriyanka) [inline panda-qa/Resources/PageObjects/PENCapabilitiesPage.robot:1]: @AkhileshKhanduri25 , Please provide the testcase/ suite name where you have executed PENCapabilities , I am unable to find where capabilities page is used

**Review by bhoomraogaripriyanka — COMMENTED** [review]
@AkhileshKhanduri25 ,please find the comments and make necessary changes .
cc: @ArchanaSaini149

File: panda-qa/Resources/PageObjects/FNArticlePage.robot
Comment (bhoomraogaripriyanka) [inline panda-qa/Resources/PageObjects/FNArticlePage.robot:1]: @AkhileshKhanduri25 ,This is not working for me , I am not able to navigate till that page,please verify previous step urls

File: panda-qa/Resources/PageObjects/MansionGlobalCapabilitiesPage.robot
Comment (bhoomraogaripriyanka) [inline panda-qa/Resources/PageObjects/MansionGlobalCapabilitiesPage.robot:1]: @AkhileshKhanduri25 , this is working fine

File: panda-qa/Resources/PageObjects/MansionGlobalPreferenceCenterPage.robot
[the remark this answers is not in the document]
Comment (bhoomraogaripriyanka) [reply panda-qa/Resources/PageObjects/MansionGlobalPreferenceCenterPage.robot:1]: @AkhileshKhanduri25 ,This is not working for me

File: panda-qa/Resources/PageObjects/MarketWatchArticlePage.robot
[the remark this answers is not in the document]
Comment (bhoomraogaripriyanka) [reply panda-qa/Resources/PageObjects/MarketWatchArticlePage.robot:1]: @AkhileshKhanduri25 , this is not working for me ,got the error of "Page  should have contained element '//div[@class="byline"]/div/div/span[text()="Catey Hill"]' but did not."

**Review by bhoomraogaripriyanka — COMMENTED** [review]
@AkhileshKhanduri25 , Please view the comments and change the code accordingly
cc: @ArchanaSaini149

File: panda-qa/Resources/PageObjects/MansionGlobalArticlePage.robot
Comment (bhoomraogaripriyanka) [inline panda-qa/Resources/PageObjects/MansionGlobalArticlePage.robot:1]: @AkhileshKhanduri25 ,this is working fine for me

File: panda-qa/Resources/PageObjects/WSJMailboxPage.robot
Comment (bhoomraogaripriyanka) [inline panda-qa/Resources/PageObjects/WSJMailboxPage.robot:1]: @AkhileshKhanduri25 ,This is not working for me

File: panda-qa/Resources/PageObjects/WSJPreferenceCenterPage.robot
Comment (bhoomraogaripriyanka) [inline panda-qa/Resources/PageObjects/WSJPreferenceCenterPage.robot:1]: @AkhileshKhanduri25 ,This is not working for me

File: panda-qa/Resources/PageObjects/WSJInflationPage.robot
Comment (bhoomraogaripriyanka) [inline panda-qa/Resources/PageObjects/WSJInflationPage.robot:1]: @AkhileshKhanduri25 ,This is not working for me

File: panda-qa/Resources/PageObjects/WSJArticlePage.robot
Comment (bhoomraogaripriyanka) [inline panda-qa/Resources/PageObjects/WSJArticlePage.robot:1]: @AkhileshKhanduri25 ,This is not working for me

**Review by bhoomraogaripriyanka — COMMENTED** [review]
@AkhileshKhanduri25 ,Please go through the comments and make changes accordingly.
cc: @ArchanaSaini149

File: panda-qa/Resources/PageObjects/PENCapabilitiesPage.robot
Comment (bhoomraogaripriyanka) [reply panda-qa/Resources/PageObjects/PENCapabilitiesPage.robot:1]: @AkhileshKhanduri25 ,this is not working for me for regression

**Review by bhoomraogaripriyanka — COMMENTED** [review]
@AkhileshKhanduri25 ,please find the comments and make changes accordingly
cc: @ArchanaSaini149

File: panda-qa/Resources/PageObjects/BarronsMailboxPage.robot
[the remark this answers is not in the document]
Comment (bhoomraogaripriyanka) [reply panda-qa/Resources/PageObjects/BarronsMailboxPage.robot:1]: @AkhileshKhanduri25 , Line 98 is not working for me . I am getting error:Page should have contained element '//div/span[text()="Barron's Stock Picks"]' but did not.

File: panda-qa/Resources/PageObjects/BarronsPreferenceCenterPage.robot
[the remark this answers is not in the document]
Comment (bhoomraogaripriyanka) [reply panda-qa/Resources/PageObjects/BarronsPreferenceCenterPage.robot:1]: @AkhileshKhanduri25 , working fine for me

File: panda-qa/Resources/PageObjects/BarronsWatchlistPage.robot
[the remark this answers is not in the document]
Comment (bhoomraogaripriyanka) [reply panda-qa/Resources/PageObjects/BarronsWatchlistPage.robot:1]: @AkhileshKhanduri25 ,Line 8 is not working for me ,I am getting error: Page should have contained element '//*[text()="Add Symbols"]' but did not. Please verify the flow once

File: panda-qa/Resources/PageObjects/WSJCapabilitiesPage.robot
Comment (bhoomraogaripriyanka) [inline panda-qa/Resources/PageObjects/WSJCapabilitiesPage.robot:1]: @AkhileshKhanduri25 , this is not working for me

**Review by bhoomraogaripriyanka — COMMENTED** [review]
@AkhileshKhanduri25 ,please find the comments of the commits
cc @ArchanaSaini149

File: panda-qa/Resources/PageObjects/BarronsArticlePage.robot
[the remark this answers is not in the document]
Comment (bhoomraogaripriyanka) [reply panda-qa/Resources/PageObjects/BarronsArticlePage.robot:1]: @AkhileshKhanduri25 ,this is not working for me ,in line 199 ,try with //span[text()="Teresa Rivas"]

File: panda-qa/Resources/PageObjects/BarronsMailboxPage.robot
[the remark this answers is not in the document]
Comment (bhoomraogaripriyanka) [reply panda-qa/Resources/PageObjects/BarronsMailboxPage.robot:1]: @AkhileshKhanduri25 ,at line 91 ,please add space before Barron's Stock Picks

File: panda-qa/Resources/PageObjects/BarronsWatchlistPage.robot
[the remark this answers is not in the document]
Comment (bhoomraogaripriyanka) [reply panda-qa/Resources/PageObjects/BarronsWatchlistPage.robot:1]: @AkhileshKhanduri25 ,Line 8 is not working for me ,I am getting error: Page should have contained element '//*[text()="Add Symbols"]' but did not. Please verify the flow once

File: panda-qa/Resources/PageObjects/PENCapabilitiesPage.robot
Comment (bhoomraogaripriyanka) [reply panda-qa/Resources/PageObjects/PENCapabilitiesPage.robot:1]: @AkhileshKhanduri25 ,this is not working for me for regression

**Review by bhoomraogaripriyanka — COMMENTED** [review]
@AkhileshKhanduri25 ,please view the comments and execute each test case after modification

cc @ArchanaSaini149

File: panda-qa/Resources/PageObjects/PENMailboxPage.robot
[the remark this answers is not in the document]
Comment (bhoomraogaripriyanka) [reply panda-qa/Resources/PageObjects/PENMailboxPage.robot:1]: Line 48 is not working
Line 64,80 is working fine

File: panda-qa/Resources/PageObjects/WSJMailboxPage.robot
Comment (bhoomraogaripriyanka) [reply panda-qa/Resources/PageObjects/WSJMailboxPage.robot:1]: Execution is failing at sigin

**Review by bhoomraogaripriyanka — COMMENTED** [review]
@AkhileshKhanduri25 ,please find the comments

cc: @ArchanaSaini149

File: panda-qa/Resources/PageObjects/PENArticlePage.robot
Comment (bhoomraogaripriyanka) [inline panda-qa/Resources/PageObjects/PENArticlePage.robot:1]: Line 78 is working fine

File: panda-qa/Resources/PageObjects/PENMailboxPage.robot
[the remark this answers is not in the document]
Comment (bhoomraogaripriyanka) [reply panda-qa/Resources/PageObjects/PENMailboxPage.robot:1]: Line 48 is not working
Line 64,80 is working fine

File: panda-qa/Resources/PageObjects/PENPreferenceCenterPage.robot
Comment (bhoomraogaripriyanka) [inline panda-qa/Resources/PageObjects/PENPreferenceCenterPage.robot:1]: Lines 52,53 are working fine

**Review by bhoomraogaripriyanka — COMMENTED** [review]
@AkhileshKhanduri25 ,Please find the comments and make the necessary changes accordingly

cc: @ArchanaSaini149

**Review by bhoomraogaripriyanka — APPROVED** [review]
@AkhileshKhanduri25 ,Working fine for me for the commits 
[68bf6d8]
[c6bb25f]
[ec533f8]

cc  @ArchanaSaini149

**Comment by AkhileshKhanduri25** [comment]
- Performed all the relevant commits required for all 5 brands: WSJ, Barrons, FN, Mansion Global and MarketWatch.
- Merging the pull request #106 as it's approved and there are no conflicts as of now.

**Pull request merged**